ext/odbc/php_odbc_includes.h

```diff
@@ -191,10 +191,15 @@ typedef struct odbc_connection {
     ODBC_SQL_CONN_T hdbc;
     char laststate[6];
     char lasterrormsg[SQL_MAX_MESSAGE_LENGTH];
-	zend_resource *res;
-	int persistent;
 } odbc_connection;
 
+typedef struct odbc_link {
+	odbc_connection *connection;
+	zend_string *hash;
+	bool persistent;
+	zend_object std;
+} odbc_link;
+
```

Comment:
I would keep ``persistent`` flag in ``odbc_connection``.
May be I didn't understand why it was moved...

Reply:
It's easier to access it in the link structure, and we don't need it in the connection. However, I can bring it back if you want. :)